exiftool/exiftool.py

```diff
@@ -101,6 +101,10 @@
 #ENCODING_LATIN1: str = "latin-1"
 
 
+class NoOutput(Exception):
```

Comment:
I think we should create a new file... I have mine as `exceptions.py`

```python
class ExifToolException(Exception):
	"""
	Generic Base class for all ExifTool error classes
	"""
```

"Exception" I think might be better than "Error"... some Exceptions may not be errors... like the NoOutput one?

then just import specific exceptions from exceptions.  I can put that stuff in... I'm still thinking about the names and exceptions exiftool needs